provider/datagen/src/transform/cldr/displaynames/language.rs

```diff
@@ -46,22 +46,46 @@ impl IterableDataProvider<LanguageDisplayNamesV1Marker> for crate::DatagenProvid
     }
 }
 
+
 /// Substring used to denote alternative region names data variants for a given region. For example: "BA-alt-short", "TL-alt-variant".
 const ALT_SUBSTRING: &str = "-alt-";
+/// Substring used to denote short display names data variants for a given language. For example: "az-alt-short".
+const ALT_SHORT_SUBSTRING: &str = "-alt-short";
+/// Substring used to denote long display names data variants for a given language. For example: "az-alt-long".
+const ALT_LONG_SUBSTRING: &str = "-alt-long";
+/// Substring used to denote menu display names data variants for a given language. For example: "az-alt-menu".
+const ALT_MENU_SUBSTRING: &str = "-alt-menu";
 
 impl From<&cldr_serde::language_displaynames::Resource> for LanguageDisplayNamesV1<'static> {
     fn from(other: &cldr_serde::language_displaynames::Resource) -> Self {
         let mut names = ZeroMap::new();
+        let mut short_names = ZeroMap::new();
+        let mut long_names = ZeroMap::new();
+        let mut menu_names = ZeroMap::new();
         for lang_data_entry in other.main.0.iter() {
             for entry in lang_data_entry.1.localedisplaynames.languages.iter() {
                 let region = String::from(entry.0);
```

Comment:
This string is unnecessary, just use `entry.0` instead.

Reply:
Had it for the last condition. But you are right that we can use entry.0 directly. Added the change.